test/game_collection_test.rb

```diff
@@ -7,7 +7,14 @@
 class GameCollectionTest < Minitest::Test
 
   def setup
-    @game_collection = GameCollection.new("./test/fixtures/games_truncated.csv")
+    locations = {
+      games: "./test/fixtures/games_truncated.csv",
+      teams: "",
+      game_teams: ""
+    }
+
+    @stat_tracker = StatTracker.from_csv(locations)
+    @game_collection = @stat_tracker.game_collection
```

Comment:
I actually think that the original code here is better for clarity's sake, we're testing GameCollection here, not StatTracker.

Reply:
Yep. I agree. Not sure what I was I was thinking 🙃